.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+**What problem is this solving?**
+
+<!--- What is the motivation and context for this change? -->
+
+**How should this be manually tested?**
+
+**Screenshots or example usage:**
```

Comment:
can you update this template to [this one](https://raw.githubusercontent.com/vtex-apps/checkout/master/.github/PULL_REQUEST_TEMPLATE.md)?